guides/getting_started_features.md

```diff
@@ -0,0 +1,31 @@
+---
+id: "getting_started_features"
+title: "Getting Started with Features"
+sidebar_label: "Features"
+---
+At this stage, you have a simple extension. If you used the blank template though, you won't have much in it. Here's where we start building out the features of your extension. While you can use vanilla Cloud Formation to build out your templates, there are also a number of features that have been built into cosmos to simplify this.
```

Comment:
do you mean vanilla CDK instead of cloud formation?

Reply:
yessss....